config/webpack/types.ts

```diff
@@ -0,0 +1,7 @@
+type EnvFile = {
```

Comment:
```suggestion
type Env = {
```

Type EnvFile is confusing with prop envFile